docs/cli/install.md

```diff
@@ -223,19 +223,19 @@ For complete documentation refer to [Package manager > Global cache](https://bun
 
 ## Minimum release age
 
-To protect against supply chain attacks where malicious packages are quickly published, you can configure a minimum age requirement for npm packages. Package versions published more recently than the specified threshold (in seconds) will be filtered out during installation.
+To protect against supply chain attacks where malicious packages are quickly published, you can configure a minimum age requirement for npm packages. Package versions published more recently than the specified threshold will be filtered out during installation.
```

Comment:
You should mention that the seconds integer format is also supported (perhaps mark it as "deprecated").

I'm still so hurt that that is not `ms`.

Reply:
Yeah i guess its useful to let people know what it does for raw numbers. But it really isn't the best to use imo